src/mlpack/tests/data/impute_test.csv

```diff
@@ -0,0 +1,3 @@
+a, 2, 3
```

Comment:
I think we need it, because this file is doing the unit test of different imputation strategies.